tests/net/lib/coap/testcase.yaml

```diff
@@ -1,5 +1,5 @@
 tests:
-  net.coap:
+  net.coap.coap:
```

Comment:
Having coap mentioned twice here looks bad. If the idea is to have three level symbol, better would be `net.lib.coap`